content/docs/reference/config/policy-and-telemetry/attribute-vocabulary/index.md

```diff
@@ -81,6 +81,7 @@ deployments will have agents (Envoy or Mixer adapters) that produce these attrib
 | `context.time`          | timestamp | The timestamp of Mixer operation. | |
 | `context.reporter.kind` | string | Contextualizes the reported attribute set. Set to `inbound` for the server-side calls from sidecars and `outbound` for the client-side calls from sidecars and gateways | `inbound` |
 | `context.reporter.uid`  | string | Platform-specific identifier of the attribute reporter. | `kubernetes://my-svc-234443-5sffe.my-namespace` |
+| `context.proxy_error_code` | string | Additional details about the response or connection from envoy. See %RESPONSE_FLAGS% in [envoy access log](https://www.envoyproxy.io/docs/envoy/latest/configuration/access_log#configuration) for more detail | `UH` |
```

Comment:
envoy -> Envoy

I'm not crazy about the fact our attributes are leaking Envoy-specific state. That stuff needs to be neutral relative to the specific proxy in use if possible.

Reply:
Reworded to make it more neutral. Thanks!